tools/state-viewer/src/apply_chain_range.rs

```diff
@@ -286,43 +286,54 @@ fn apply_block_from_range(
             apply_result.trie_changes.state_changes().len(),
         ),
     );
-    progress_reporter.inc_and_report_progress(apply_result.total_gas_burnt);
-
-    if mode == ApplyRangeMode::Benchmarking {
-        // Compute delta and immediately apply to flat storage.
-        let changes =
-            FlatStateChanges::from_state_changes(apply_result.trie_changes.state_changes());
-        let delta = near_store::flat::FlatStateDelta {
-            metadata: near_store::flat::FlatStateDeltaMetadata {
-                block: BlockInfo {
-                    hash: block_hash,
-                    height: block.header().height(),
-                    prev_hash: *block.header().prev_hash(),
+    progress_reporter.inc_and_report_progress(height, apply_result.total_gas_burnt);
+
+    match (mode, storage) {
+        (ApplyRangeMode::Benchmark, _) => {}
```

Comment:
Just for my information, what's this used for and who are the owners? I don't have enough context on this to properly review but I'll be happy to stamp this PR anyway if you're confident with it. 

Previously the deltas were also applied in this mode, just sanity checking - did you mean to not do it in this mode? 

nit: Can you add a short comment for this match statement? Something about what and why happens here.